src/core/core.tooltip.js

```diff
@@ -557,14 +563,34 @@ module.exports = function(Chart) {
 				}
 			}
 
+			this.applyStroke(tooltipPoint, vm, ctx, opacity);
```

Comment:
why is apply stroke called here? what is affected by the `ctx.stroke()` at the end of that method?